spec/models/post.spec.js

```diff
@@ -15,6 +15,12 @@ describe("Post model", () => {
     expect(post.message).toEqual("some message");
   });
 
+
+  it("has a message with multiple lines", () => {
+    var post = new Post({ message: 'This is the first line.\r\n' + 'This is the second line.\r\n' });
+    expect(post.message).toEqual('This is the first line.\r\n' + 'This is the second line.\r\n');
+  });
```

Comment:
I'm not entirely sure why this is presented as two strings connected together. Does this reflect how the frontend will interact with the form submission?